### ISSUE TITLE
Problem with update accessories in iOS since I update the add-on to 2.4.0

### ISSUE THREAD

**Issue by 5chacha5** [hassio-addons/addon-homebridge#85]
# Problem/Motivation

Since I update this add-on to 2.4.0, the update of accessories under iOS is very very long or impossible.

## Expected behavior

> Same speed of the add-on 2.3.0

## Actual behavior

> error during the update in iOS

Can you help me ?

**Comment by AndBobsYourUncle** [comment]
I also submitted a ticket about this. It appears that certain types of accessories are the potential cause of the problem.

https://github.com/hassio-addons/addon-homebridge/issues/84

For me, it was the climate accessory for my Nest. When I excluded that from my HomeBridge setup, the accessories magically started getting updates again.

**Comment by 5chacha5** [comment]
that's the same for me, it's the climate accessory ! but I use broadlink custom component to control my air cooler and radiator and I discover a problem with my radiator, I solve it and that work now.

thanks for your answer

**Comment by AndBobsYourUncle** [comment]
Yeah, it seems we'll just have to wait for the next update to see if they fix the climate thing. It's kind of annoying, but at least I can control my Nest through the Nest app in the meantime? 🤷‍♂️

**Comment by CroVlado** [comment]
Have you tried adding the nest to hassio and then exporting it to homebridge as a device instead of using homebridge-nest plugin?

**Comment by AndBobsYourUncle** [comment]
@CroVlado 

My nest is added straight into Home Assistant. I then have this add-on. This worked in version 2.3.0, but ever since 2.4.0 of this add-on, it stopped working.

**Comment by CroVlado** [comment]
You can delete the nest plugin and just use the homebridge_visible command in customize yaml to have it show up in home

**Comment by AndBobsYourUncle** [comment]
I’m not using the nest plugin. I have the entity in Home Assistant setup and working properly. This addon then adds two accessories in HomeKit, one for the fan, and one for the climate controls.

The problem is that since this update, both accessories in HomeKit never finish updating. In the Home app is basically times out trying to get the current state.

**Comment by CroVlado** [comment]
I got ya. Have you tried replacing that with homebridge-nest and seeing if it creates the same issues? 
I use homebridge-nest until the whole HomeKit stuff comes along better in HA. It works well for me.

**Comment by nashlib** [comment]
For me it’s a full on crash when I add BroadlinkRM Homebridge plug-in 
When I go into the logs I see that it’s broadlink plugin that is not fully loading. When I remove that plug in and the corresponding entry on the config file things a back to normal.

**Comment by jsaunders92** [comment]
I'm having the same issue here, but with my Xiaomi Air Purifier

**Comment by nashlib** [comment]
Hi mates,
I was trying to find a solution to the  problem as this was working fine before. I think it’s more to do with my HomePod. 

As a process of elimination:

I removed my HomePod
Setup the Apple TV 4 as the hub
Then I restarted Homebridge
Checked on both home network and on 4G
Re added HomePod
Updated HomePod to 11.3
Re tried homebridge
Now it seems to work and has been stable for past 5 days.
Hope this helps.
